### ISSUE TITLE
Unable to install elastic search set up

### ISSUE THREAD

**Issue by suresh-im** [elastic/homebrew-tap#157]
brew install elastic/tap/elasticsearch-full
Error: elastic/tap/elasticsearch-full: Calling plist_options is disabled! Use service.require_root instead.
Please report this issue to the elastic/tap tap (not Homebrew/brew or Homebrew/homebrew-core), or even better, submit a PR to fix it:
  /usr/local/Homebrew/Library/Taps/elastic/homebrew-tap/Formula/elasticsearch-full.rb:68

**Comment by rohanmendon** [comment]
getting the issue

**Comment by harryjackson** [comment]
I don't think this project is maintained anymore. To install elastic now you need to use the following

https://www.elastic.co/guide/en/elasticsearch/reference/8.10/install-elasticsearch.html

I wish they'd put a warning on the README.

**Comment by wflanagan** [comment]
Yes, I agree. I have jumped to OpenSearch for development. Probably will migrate production there shortly.

**Comment by kevinhq** [comment]
coming to this page from google search result, with exactly same problem. i wish they put the updates on readme.
